src/test/java/io/reactivex/internal/operators/flowable/FlowablePublishTest.java

```diff
@@ -941,4 +941,365 @@ protected void subscribeActual(Subscriber<? super Integer> s) {
         ref.get().add(new InnerSubscriber<Integer>(new TestSubscriber<Integer>()));
         ref.get().remove(null);
     }
+
+    @Test
+    @Ignore("publish() keeps consuming the upstream if there are no subscribers, 3.x should change this")
```

Comment:
Should we document this somewhere in one of the 3.x issues?

Reply:
Posted as #5899.